docs/source/configuration/slots.md

```diff
@@ -134,3 +134,91 @@ It accepts a list of possible content types.
 You can create your own predicate helpers to determine whether your slot component should render.
 The `SlotRenderer` will pass down the current `content` and the `pathname` into your custom predicate helper.
 You can also tailor your own `SlotRenderer`s, or shadow the original `SlotRenderer`, to satisfy your requirements.
+
+## Manage registered slots and slot components
+
+### `getSlotComponents`
+
+It returns the list of components registered per slot.
+This is useful to debug what is registered an in what order, and later change the order, if needed.
+This is the signature:
+
+```ts
+config.getSlotComponents(slot: string): string[]
+
+```
+
+`slot`
+:   The name of the slot, where the slot components are stored.
+
+### `reorderSlotComponent`
+
+It reorders the list of components registered per slot.
+This is the signature:
+
+```ts
+config.reorderSlotComponent(slot: string, name: string, position: number): void
+```
+
+`slot`
+:   The name of the slot, where the slot components are stored.
+
+`name`
+:   The name of the slot component we want to reorder.
+
+`position`
+:   The destination position in the registered list of slot components that we want to move the slot component.
+
+
+### `getSlotComponent`
+
+It returns the list of registered components per slot component name.
+This is useful to debug what is registered an in what order, and later remove a registration, if needed.
+This is the signature:
+
+```ts
+config.getSlotComponent(slot: string, name: string): SlotComponent[]
+```
+
+`slot`
+:   The name of the slot, where the slot components are stored.
+
+`name`
+:   The name of the slot component we want to retrieve.
+
+### `unRegisterSlotComponent`
+
+It removes a registration for a specific component, given its registration position.
+This is the signature:
+
+```ts
+config.unRegisterSlotComponent(slot: string, name: string, position: number): void
+```
+
+`slot`
+:   The name of the slot, where the slot components are stored.
+
+`name`
+:   The name of the slot component inside it's the component we want to unregister.
+
+`position`
+:   The component position that we want to remove in the slot component registration. Use `getSlotComponent` to find out the position of the registered component that you want to remove.
```

Comment:
Is the position zero- or one-indexed? I assume zero, but want to verify.

Reply:
zero